pom.xml

```diff
@@ -77,7 +77,7 @@
     <jackson.version>1.9.11</jackson.version>
     <jackson.package>org.codehaus.jackson</jackson.package>
     <shade.prefix>shaded.parquet</shade.prefix>
-    <hadoop.version>1.1.0</hadoop.version>
+    <hadoop.version>2.3.0</hadoop.version>
```

Comment:
agreed, I believe there is compatibility code included, but I don't know if something has been developed to allow for cross-version automated testing. I'll try to take a closer look at the tests added and work on a solution if there isn't one already.